server/src/ClientToDBAPI/ContactedPeople/mainRoute.ts

```diff
@@ -0,0 +1,14 @@
+import { Router, Request, Response } from 'express';
+import { graphqlRequest } from '../../GraphqlHTTPRequest';
+import { GET_CONTACTED_PEOPLE } from '../../DBService/ContactedPeople/Query';
+
+const ContactedPeopleRoute = Router();
+
+ContactedPeopleRoute.get('/:investigationId', (request: Request, response: Response) =>
+    graphqlRequest(GET_CONTACTED_PEOPLE, response.locals, {investigationId: parseInt(request.params.investigationId)})
+        .then((result: any) => response.send(result))
```

Comment:
maybe you should parse the result to a clearer object and send the parsed object instead

Reply:
For understanding the structure of the result?

Reply:
hello my lovely friends, 
the result is an array of items 
i can add parsing here, but because its an array and not a single object i need to use map/loop and an extra array for the parsed objects - i would like to know what the reason for parsing the result, to know if its worth it. 
i saw that the parsing not happening in most of the routes. 
again, if you see it necessary i will change it

Reply:
<img width="892" alt="Screen Shot 2020-09-22 at 16 07 09" src="https://user-images.githubusercontent.com/24268917/93886142-bc034100-fced-11ea-9093-9d9f4fb58649.png">
this is the parsing that im talking about, if its better for your opinion so i will commit it